discoverable-client/src/main/resources/application.yml

```diff
@@ -10,6 +10,7 @@ logging:
         com.netflix.discovery.shared.transport.decorator.RedirectingEurekaHttpClient: OFF
         com.netflix.discovery.DiscoveryClient: OFF
         org.springframework.boot.web.embedded.tomcat.TomcatWebServer: INFO
+        com.ca.mfaas.client.api: TRACE
```

Comment:
Do we want to have something set to trace?

Reply:
I forgot to delete. It was created for development purpose.